test_s3.py

```diff
@@ -1,29 +1,70 @@
 import boto3
+import sys
+import logging
+import argparse
+import yaml
 
-# Create S3
+bucket_name = 'test-bucket-boto-mzol'
+s3resource = boto3.resource('s3')
+bucket = s3resource.Bucket(bucket_name)
 s3 = boto3.client('s3')
 
-bucket_name = 'test-bucket-boto-mzol'
-print('Creating new bucket with name: Test-Bucket-boto-mzol')
-s3.create_bucket(Bucket=bucket_name)
+"""Create S3"""
+def CreateS3(bucket_name,logger):
```

Comment:
method names shouldn't be camel case. https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Done